packages/now-node-server/package.json

```
@@ -8,8 +8,8 @@
     "directory": "packages/now-node-server"
   },
   "dependencies": {
-    "@now/node-bridge": "1.2.0-canary.3",
-    "@zeit/ncc": "0.18.5",
+    "@now/node-bridge": "^1.2.0-canary.3",
```

```suggestion
    "@now/node-bridge": "1.2.0-canary.3",
```

Please revert this one back to exact version